corehq/apps/receiverwrapper/tests/test_submissions.py

```diff
@@ -166,6 +171,30 @@ def list_attachments(form):
             list_attachments(new_form),
             [("file", b"text file"), ("image", b"other fake image")])
 
+    def test_submit_invalid_attachment_size(self):
+        file_data = BytesIO(b"a" * (settings.MAX_UPLOAD_SIZE_ATTACHMENT + 1))
+        expected_error = AttachmentSizeTooLarge()
```

Comment:
nit: Looks like this is now just getting used at one place later for status code, you could probably just get rid of it now.